packages/core/src/RenderingEngine/RenderingEngine.ts

```diff
@@ -1023,22 +1044,22 @@ class RenderingEngine implements IRenderingEngine {
     _xOffset: number
   ): ViewportDisplayCoords {
     const { canvas } = viewport;
-    const { clientWidth, clientHeight } = canvas;
+    // const { clientWidth, clientHeight } = canvas;
     const devicePixelRatio = window.devicePixelRatio || 1;
-    const height = clientHeight * devicePixelRatio;
-    const width = clientWidth * devicePixelRatio;
+    // const sHeight = clientHeight * devicePixelRatio;
+    // const sWidth = clientWidth * devicePixelRatio;
+    const { width: sWidth, height: sHeight } = canvas;
```

Comment:
This is the primary fix for the size differences - using the actual width/height makes things actually work